src/manifest.rs

```diff
@@ -229,13 +241,24 @@ impl str::FromStr for Manifest {
         let mut parser = toml::Parser::new(&input);
 
         parser.parse()
-              .ok_or(parser.errors.pop())
+              .ok_or(format_parse_error(parser))
               .map_err(Option::unwrap)
               .map_err(From::from)
               .map(|data| Manifest { data: data })
     }
 }
 
+fn format_parse_error(mut parser: toml::Parser) -> Option<ManifestError> {
+    match parser.errors.pop() {
```

Comment:
If I'm reading this correctly, you are only looking at the last error. What other parser errors can there be? Should we show those as well?